src/lib/sql/sql_pipeline_statement.hpp

```diff
@@ -134,8 +133,10 @@ class SQLPipelineStatement : public Noncopyable {
   // Delete temporary tables
   const CleanupTemporaries _cleanup_temporaries;
 
-  std::optional<std::string> _warning_message;
-  bool _is_transaction_statement;
+  std::optional<std::string> _warning_message = std::nullopt;
+
+  // Might be the Statement's own transaction context, or the one shared by all Statements in a Pipeline
```

Comment:
Own transaction context **(if in auto-commit mode)**?

Reply:
Done.